niceman/distributions/tests/test_conda.py

```diff
@@ -81,6 +81,9 @@ def test_conda_manager_identify_distributions(get_conda_test_dir):
         "/sbin/iptables",
         os.path.join(test_dir, "minimal_pymodule")}
 
+    assert distributions.platform in ["linux-64", "osx-64"], \
```

Comment:
this would fail whenever we run tests on 32bit, etc whenever we get there to install 32bit conda in 32bit env (time will come).  So why not to make it more specific if e.g. in this whole setup we assume that conda should match our native env, it should match one to one (after mapping) then to output of the local python's `platform.architecture()`

Reply:
Good point.  I found the python to conda platform mapping that conda does in its source code: https://github.com/conda/conda/blob/301133cb8c804730c218ad8e47f116a8b1af51fc/conda/models/enums.py .  So I created a function to map python platforms to conda platforms and then updated our unit test to make sure that the detected platform is what's expected for the current environment.